cdc/api/v2/changefeed.go

```diff
@@ -294,6 +297,71 @@ func (h *OpenAPIV2) GetChangeFeedMetaInfo(c *gin.Context) {
 	c.JSON(http.StatusOK, toAPIModel(info))
 }
 
+// ResumeChangefeed handles update changefeed request.
+func (h *OpenAPIV2) ResumeChangefeed(c *gin.Context) {
+	if !h.capture.IsOwner() {
+		api.ForwardToOwner(c, h.capture)
+		return
+	}
+
+	ctx := c.Request.Context()
+	changefeedID := model.DefaultChangeFeedID(c.Param(apiOpVarChangefeedID))
+	err := model.ValidateChangefeedID(changefeedID.ID)
+	if err != nil {
+		_ = c.Error(cerror.ErrAPIInvalidParam.GenWithStack("invalid changefeed_id: %s",
+			changefeedID.ID))
+		return
+	}
+
+	var startTs uint64
+	startTsStr, ok := c.GetQuery("start_ts")
+	if ok {
```

Comment:
so the start_ts is not required?

Reply:
> so the start_ts is not required?

Yes, it's not required. If this param is not supplied, we will follow the old resume behavior which continues from where it left off